internal/controlplane/common.go

```diff
@@ -49,3 +56,59 @@ func getNameFilterParam(name string) sql.NullString {
 		Valid:  name != "",
 	}
 }
+
+// getRemediationURLFromMetadata returns the "remediation URL". For now, this is
+// the URL link to the PR
+func getRemediationURLFromMetadata(data []byte, repoSlug string) (string, error) {
+	// If no data, it means no PR is tracked.
+	// So no error and we return an empty string.
+	if len(data) == 0 {
+		return "", nil
+	}
+	prData := &struct {
+		Number int `json:"pr_number"`
+	}{}
+
+	if err := json.Unmarshal(data, prData); err != nil {
+		return "", fmt.Errorf("unmarshalling pull request metadata: %w", err)
+	}
+
+	if repoSlug == "" && prData.Number == 0 {
+		return "", fmt.Errorf("no repository defined")
+	}
+
+	// No pull equest found
+	if prData.Number == 0 {
+		return "", nil
+	}
+
+	if !validRepoSlugRe.MatchString(repoSlug) {
+		return "", fmt.Errorf("invalid repository slug")
+	}
+
+	return fmt.Sprintf("%s/%s/pull/%d", githubURL, repoSlug, prData.Number), nil
+}
+
+// getAlertURLFromMetadata is a helper function to get the alert URL from the alert metadata
+func getAlertURLFromMetadata(data []byte, repo string) (string, error) {
```

Comment:
we call this `repoSlug` above.  Is there a difference?  Does this (or the other) include the owner part?

Reply:
I've evened the param names. Now both are `repoSlug`.